contracts/BosonRouter.sol

```diff
@@ -50,21 +50,21 @@ contract BosonRouter is
      * @notice Acts as a modifier, but it's cheaper. Checking if a non-zero address is provided, otherwise reverts.
      */
     function notZeroAddress(address tokenAddress) private pure {
-        require(tokenAddress != address(0), "0_A"); //zero address
+        require(tokenAddress != address(0), "0A"); //zero address
```

Comment:
Are these changes needed for this PR?

Reply:
Yes. I think this was discussed last week (Thursday) on the call we had.